src/matrixHelpers/FormioLoader/index.js

```diff
@@ -45,16 +43,19 @@ const initFormioInstance = (formioElem, opts) => {
   /*
    * load formio form
    */
-  Formio.createForm(
-    formioElem,
-    formUrl,
-    // form,
-    {
-      ...createFormOptions,
-      formio,
-      namespace: formio.options.namespace,
-    }
-  ).then((wizard) => {
+  const defaultOptions = {
+    ...createFormOptions,
+    formio,
+    namespace: formio.options.namespace,
+  };
+  const combinedOptions = {
+    ...defaultOptions,
+    // combine with webhook options
+    ...(window.formioCreateFormOptions
+      ? window.formioCreateFormOptions({ ...opts, defaultOptions })
```

Comment:
I guess this does not handle two 'scripts' trying to set this function? 
How could we  allow that to occur?

Also is this per form creation or only at start.

i.e. if i 'replace' the formioCreateFormOptions function and then manually load a new form.

Reply:
yes it will overwrite each other if set in 2+ different scripts... i probably need to to use another approach to support that

with either approach the hook function only need to defined once, but no harm to define multiple times

Reply:
Also;
would you need to have your script 'init' prior to the loader kicking off 
OR 
before it reaches the div data element/code that creates the form?

Reply:
the hook functions could be placed anywhere in the page, but need to be defined before the the loader is initiated

ie. 
```
<div data-formio ... />
<script>
window.formioCreateFormOptionsHook = ()=> { return blah } OR window.formioCreateFormOptions["envUrl"]["projectId"]["formId"] (new approach for specific form)
window.formioCreateFormControllerHook = ()=> { blah() } OR window.formioCreateFormController["envUrl"]["projectId"]["formId"] (new approach for specific form)
FormioLoader.initFormio() OR FormioLoader.initFormioInstance()
</script>
```
or 
```
<div data-formio ... />
<script>
window.formioCreateFormOptionsHook = ()=> { return blah } OR window.formioCreateFormOptions["envUrl"]["projectId"]["formId"] (new approach for specific form)
window.formioCreateFormControllerHook = ()=> { blah() } OR window.formioCreateFormController["envUrl"]["projectId"]["formId"] (new approach for specific form)
</script>
<script src=".../formio-script.js" />
```

Reply:
may need that in the doc's